.github/workflows/test-suite.yml

```diff
@@ -131,18 +139,24 @@ jobs:
       run: make test-slasher
   debug-tests-ubuntu:
     name: debug-tests-ubuntu
-    runs-on: ubuntu-22.04
```

Comment:
I've swapped from `22.04` to `latest` here. If CI fails I'll switch it back.

Reply:
`latest` seems to work, so I'll leave it as that.

Reply:
Oh, it's probably just working on `latest` because we're using self-hosted runners. I might just leave it on latest anyway. It was set to `22.04` in https://github.com/sigp/lighthouse/pull/3354 and I'm not exactly sure why.

Reply:
We should be good-  `latest` is the same as `22.04` now
https://github.com/actions/runner-images/blob/main/README.md?plain=1#L24

looks like debug tests didn't work in `latest` (=`20.04`) at the time of #3354 and it was switched to use 22.04 to make the job pass